src/js/components/covid19/recipient/map/MapWrapper.jsx

```diff
@@ -7,7 +7,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { uniq, cloneDeep } from 'lodash';
 
-import * as MapHelper from 'helpers/mapHelper';
+import * as Covid19Helper from 'helpers/covid19Helper';
```

Comment:
we can change this to just `import { calculateCovidMapRange } from 'helpers/covid19Helper';`

we do not need to import all the function in covid19Helper.